src/styles/index.scss

```diff
@@ -1,9 +1,6 @@
-// Global styles entry point, these are NOT BEM'ed
-
+// Vendor CSS
 @import 'vendor/normalize';
+@import url("//cloud.typography.com/6240112/781206/css/fonts.css");
```

Comment:
What project does this use? Would like to avoid loading all those Gotham SSm font variants that are loaded in prod...